Picture.py

```diff
@@ -358,14 +435,19 @@ def OnInit(self):
                 self.frame.Show()
                 return True
 
-        # Need wx app to run in separate thread or process.  It seems
-        # that starting a wxPython GUI in a worker thread is not a good idea
-        # so the easiest thing to do is just fork a new process.
-        pid = os.fork()
-        if pid == 0:
+        def doShow():
```

Comment:
It also looks like you put in a good bit of support for wx images, but it may make more sense to do more of the wx adaptation in a separate branch. I noticed was that lines 438-451 it looks like its trying to fix the issue we've been having of not being able input any other commands while showing an image. Does this solve that issue with show? If not then could you explain it help me to understand it more clearly.

Reply:
The code here does work on Linux and should work on a mac, but apparently does not work on Windows.  Fixing this is very important but will need a different approach